build.gradle

```diff
@@ -206,7 +206,7 @@ ext {
 
 dependencies {
 
-  implementation group: 'com.github.hmcts', name: 'et-data-model', version:'3.18.30'
+  implementation group: 'com.github.hmcts', name: 'et-data-model', version:'6fe9a86e8c'
```

Comment:
Is it not easier to track the version number of et-data-model if we use the serially number simple version tags?

Reply:
I think because it's not going into master, it should be ok to use this. There could be further changes to the data model which is why it's not a version number